docs/docs/events.md

```diff
@@ -5,21 +5,41 @@
 Called once the app is launched. This event is used to set the Application's initial layout - after which the app is ready for user interaction.
 
 ```js
-Navigation.events().registerAppLaunchedListener(() => {
+appLaunchedListener = Navigation.events().registerAppLaunchedListener(() => {
 
 });
 ```
 
+RNN automatically unsubscribes components when they unmount, therefore unsubscribing isn't actually mandatory if you subscribed in `componentDidMount`.
+
+But you can use the following method to unsubscribe manually.
+
+```js
+appLaunchedListener.remove();
+```
+
 ## componentDidAppear
 Called each time this component appears on screen (attached to the view hierarchy)
 
 ```js
 class MyComponent extends Component {
   constructor(props) {
     super(props);
-    Navigation.events().bindComponent(this);
+    
+    this.navigationEventListener = null;
```

Comment:
You don't need the whole constructor anymore so you can remove them in components 👍

Reply:
@henrikra Gotcha!

Reply:
Done